djangogirls/urls.py

```diff
@@ -12,5 +12,6 @@
     # Regular links:
     url(r'^admin/', include(admin.site.urls)),
     url(r'^pages/', include('django.contrib.flatpages.urls')),
+    url(r'^jobs/?', include('jobs.urls', namespace='jobs')),
```

Comment:
well, I've put it as otherwise if we type djangogirls.org/jobs (without a slash), we will be redirected to the events page and that is a bit confusing. And if we had url(r'^jobs', include('jobs.urls', namespace='jobs')); typing djangogirls.org/jobs/ would redirect us to the events page. Is this not a problem?